plans/tier1.fmf

```diff
@@ -122,10 +135,18 @@ description+: |
                   when: distro == centos-7, oracle-7
             environment+:
                 CONVERT2RHEL_SKIP_KERNEL_CURRENCY_CHECK: 1
+
```

Comment:
The exact line you have referenced should be removed altogether.
It's just a leftover from poorly modified test (the envar was changed in the test, because it was removed from accepted variants, but the comment stayed the same).
I can add the mention of what is the envar supposed to bypass at least

Reply:
Ok, thanks for the explanation